kostyor/conf.ini

```diff
@@ -1,3 +1,4 @@
 [global]
 host = 127.0.0.1
 port = 5000
+discovery_methods = first,second
```

Comment:
There are 2 types of discovery methods - default and non-default. Default
methods are always there and you can use them in every openstack setup for
every platform. Non-default methods are platform and setup-specific and
should be listed in the config file

29 июля 2016 г. 11:06 пользователь "Julia Varlamova" <
notifications@github.com> написал:

> In kostyor/conf.ini
> <https://github.com/sc68cal/Kostyor/pull/12#discussion_r72755268>:
>
> > @@ -1,3 +1,4 @@
> >  [global]
> >  host = 127.0.0.1
> >  port = 5000
> > +discovery_methods = first,second
>
> Dima, why did you add 'first' and 'second' discovery methods? Why didn't
> you add 'openstack' method, which is already implemented in
> inventory/discover.py?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sc68cal/Kostyor/pull/12/files/5c8dab03b11bf5d0f61301bbb3595efe5be574b5#r72755268>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEzEDU3caTMiXIu-mPd36Zq3xpN3-Uptks5qabR9gaJpZM4JQxHZ>
> .
>